Detectors/Base/include/DetectorsBase/Utils.h

```diff
@@ -42,11 +44,11 @@ namespace AliceO2 {
         c = cos(ang);
```

Comment:
I know it's not part of your change, but why reimplementing `sincosf` (which actually exists in C) to be less precise (due to the cast to double) and slower (due to the fact you calculate sin and cos separately)??